packages/taxdat/R/run_stan_helpers.R

```diff
@@ -35,16 +40,21 @@ get_stan_parameters <- function(config,
                          time_effect_autocorr = time_effect_autocorr,
                          censoring = censoring,
                          use_weights = use_weights,
-                         overdispersion = overdispersion,
-                         use_rho_prior = use_rho_prior,
+                         overdispersion = overdispersion, #this is not included in the wiki
```

Comment:
We think these two could be deleted but we should double check that they are not referenced anywhere else in the pipeline

Reply:
use_rho_prior is actually used in the stan model, so we should keep this. 
overdispersion was remove throughout.